filament/backend/src/metal/MetalDriver.mm

```diff
@@ -193,7 +193,8 @@
             getColorTexture(), getDepthTexture(), color.level, depth.level);
 
     ASSERT_POSTCONDITION(
-            !stencil.handle && !(targetBufferFlags & TargetBufferFlags::STENCIL),
+            !stencil.handle &&
+            !(targetBufferFlags & TargetBufferFlags::STENCIL),
```

Comment:
Should we use `none` here?

Reply:
we could, I added none() for completeness with any(). But none() and operator! are the same.